FlowCrypt/Core/CoreHost.swift

```diff
@@ -54,14 +55,15 @@ final class CoreHost: NSObject, CoreHostExports {
         return Cryptor(operation: .decrypt, algorithm: .aes, mode: .CFB, padding: .NoPadding, key: key, iv: iv).update(byteArray: ct)!.final()!
     }
 
-    // rsa verify is used by OpenPGP.js during decryption as well to figure out our own key preferences
-    // this slows down decryption the first time a private key is used in a session because bn.js is slow
-    // Using GMP C library modular exponentiation reduces rsa4096 verify time from 800ms to 40ms
     func verifyRsaModPow(_ base: String, _ exponent: String, _ modulo: String) -> String {
-        // only supported on arm64 because was not able to build it for other platforms yet
-        // in fact, I'm no longer able to build any working library other than the one I've built before
-        // when not supported, the function returns empty string, and JS falls back on bn.js
```

Comment:
This comment is still important (last line), but it could start with "If there is an error parsing provided numbers"